### ISSUE TITLE
stack dependencies in cascading configs

### ISSUE THREAD

**Issue by RolandOtta** [Sceptre/sceptre#946]
assuming i have the following project structure

```
.
├── config.yaml
├── prod
│   ├── config.yaml
│   ├── vpc.yaml
│   └── vpcPeering.yaml
├── test
│   ├── config.yaml
│   ├── vpc.yaml
│   └── vpcPeering.yaml
├── vpc.yaml
└── vpcPeering.yaml
```

i would like to define a dependency in vpcPeering.yaml to vpc.

at the moment the only way i could get this work is to define the dependencies in both ./test/vpcPeering.yaml and ./prod/vpcPeering.yaml

by adding 

```
dependencies:
    - {{ stack_group_config.profile }}/vpc.yam
```

is there a smarter way to do this. ideally i want to configure the dependency just 1 time in ./vpcPeering.yaml but i was not able to do so.

**Comment by zaro0508** [comment]
I do not believe cascading dependencies are supported and I can't think of a way to make sceptre do that without code changes.
